### PR TITLE
hardening/630_update_naming_conventions_sink_docs

### PR DESCRIPTION
**Pull request by frbattid**
* Partially implements issue #630 
* No tests are required
* Assignee @pcoello25

### CODE REVIEW

File: doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_ckan_sink.md [hunk 1]
```diff
@@ -35,31 +44,58 @@ This is done at the Cygnus Http listeners (in Flume jergon, sources) thanks to [
...
+Please observe if the notified/defaulted FIWARE service path is the root one, i.e. `/`, then the package/dataset name is equals to the organization name.
+
+The concatenation character is the underscore, `_`.
+
+[Top](#top)
+
+####<a name="section1.2.3"></a>Resources naming conventions
+CKAN resources follow a single data model (see the [Configuration](#section2.1) section for more details), i.e. per entity. Thus, a resource name always take the entity ID and type, concatenated by the underscore character, `_`. Such a name is already given in the `notified_entities`/`grouped_entities` header values (depending on using or not the grouping rules, see the [Configuration](#section2.1) section for more details) within the Flume event.
+
+It must be noticed a CKAN Datastore (and a viewer) is also created and associated to the resoruce above. This datastore, which in the end is a PostgreSQL table, will hold the persisted data.
```
Comment (pcoello25) [inline doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_ckan_sink.md]: resoruce -> resource

File: doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_ckan_sink.md [hunk 2]
```diff
@@ -35,31 +44,58 @@ This is done at the Cygnus Http listeners (in Flume jergon, sources) thanks to [
...
-    * `attrName`: Notified attribute name.
-    * `attrType`: Notified attribute type.
-    * `attrValue`: In its simplest form, this value is just a string, but since Orion 0.11.0 it can be Json object or Json array.
-    * `attrMd`: It contains a string serialization of the metadata array for the attribute in Json (if the attribute hasn't metadata, an empty array `[]` is inserted).
-* `column`: A single row is upserted for all the notified context attributes. This kind of row will contain two fields per each entity's attribute (one for the value, called `<attrName>`, and other for the metadata, called `<attrName>_md`), plus four additional fields:
-    * `recvTime`: UTC timestamp in human-redable format ([ISO 8601](http://en.wikipedia.org/wiki/ISO_8601)).
-    * `fiwareServicePath`: The notified one or the default one.
-    * `entityId`: Notified entity identifier.
-    * `entityType`: Notified entity type.
+* `recvTime`: UTC timestamp in human-redable format ([ISO 8601](http://en.wikipedia.org/wiki/ISO_8601)).
```
Comment (pcoello25) [inline doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_ckan_sink.md]: redable -> readable

File: doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_dynamodb_sink.md [hunk 3]
```diff
@@ -29,41 +33,72 @@ Next sections will explain this in detail.
...
+
+[Top](#top)
+
+####<a name="section1.2.3"></a>Row-like storing
+Regarding the specific data stored within the datastore associated to the resource, if `attr_persistence` parameter is set to `row` (default storing mode) then the notified data is stored attribute by attribute, composing an insert for each one of them. Each insert contains the following fields:
 
-* `table-by-destination`. A table named as the concatenation of `<fiware-service>_<fiware_servicePath>_<destination>` is created (if not yet existing). These values are notified as http headers.
-* `table-by-service-path`. A table named as the `<fiware-service>_<fiware-servicePath>` is created (if not yet existing). These values are notified as http headers.
+* `recvTimeTs`: UTC timestamp expressed in miliseconds.
+* `recvTime`: UTC timestamp in human-redable format ([ISO 8601](http://en.wikipedia.org/wiki/ISO_8601)).
```
Comment (pcoello25) [inline doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_dynamodb_sink.md]: redable -> readable

File: doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_dynamodb_sink.md [hunk 4]
```diff
@@ -29,41 +33,72 @@ Next sections will explain this in detail.
...
-    * `attrName`: Notified attribute name.
-    * `attrType`: Notified attribute type.
-    * `attrValue`: In its simplest form, this value is just a string, but since Orion 0.11.0 it can be Json object or Json array.
-    * `attrMd`: It contains a string serialization of the metadata array for the attribute in Json (if the attribute hasn't metadata, an empty array `[]` is inserted).
-* `column`: A single data item is added for all the notified context attributes. This kind of item will contain two fields per each entity's attribute (one for the value, named `<attrName>`, and other for the metadata, named `<attrName>_md`), plus four additional fields:
-    * `recvTime`: UTC timestamp in human-redable format ([ISO 8601](http://en.wikipedia.org/wiki/ISO_8601)).
-    * `fiwareServicePath`: The notified one or the default one.
-    * `entityId`: Notified entity identifier.
-    * `entityType`: Notified entity type.
+* `recvTime`: UTC timestamp in human-redable format ([ISO 8601](http://en.wikipedia.org/wiki/ISO_8601)).
```
Comment (pcoello25) [inline doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_dynamodb_sink.md]: Same redable :smiley:

Comment (frbattid) [inline doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_dynamodb_sink.md]: Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1042/commits/3913c56cb58a1dea3f077c77817cdce9080c530f

File: doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_dynamodb_sink.md (hunk 3 above)
Comment (frbattid) [inline doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_dynamodb_sink.md]: Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1042/commits/3913c56cb58a1dea3f077c77817cdce9080c530f

File: doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_ckan_sink.md (hunk 2 above)
Comment (frbattid) [inline doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_ckan_sink.md]: Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1042/commits/3913c56cb58a1dea3f077c77817cdce9080c530f

File: doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_ckan_sink.md (hunk 1 above)
Comment (frbattid) [inline doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_ckan_sink.md]: Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1042/commits/3913c56cb58a1dea3f077c77817cdce9080c530f

**Comment by pcoello25** [comment]
LGTM

**Pull request merged**